applications/plugins/Compliant/odesolver/CompliantPseudoStaticSolver.inl

```diff
@@ -85,8 +86,7 @@ void CompliantPseudoStaticSolver<CompliantOdeSolver>::solve(const core::ExecPara
 
     d_lastVelocity.setValue(lastVelocity);
 
-    if( printLog ) sout<<i+1<<" iterations"<<sendl;
-
+    msg_info() << tmpStr.str() << i+1 << " iterastions";
```

Comment:
msg_info_when(printLog) ?

Reply:
no, this is not needed.. msg_info() is alread having a test on printlog by default.